### PR TITLE
Added EVS DB Table and Schema entities

### PR DESCRIPTION
**Pull request by supriyakolate**
Added apprenticeship schema and all the table entities designed under EVS DB logical design

### CODE REVIEW

File: src/SFA.DAS.EmploymentVerification.Database/Tables/LogError.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE [dbo].[LogError]
+(
+  ErrorLogId BIGINT NOT NULL Identity(1,1) Constraint PK_LogError PRIMARY KEY,
```
Comment (abroodie) [inline src/SFA.DAS.EmploymentVerification.Database/Tables/LogError.sql:3]: Rename column to match the table name

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+(
+  ScheduledECId BIGINT NOT NULL Identity(1,1) Constraint PK_ScheduledEmploymentVerification PRIMARY KEY,
+  CorrelationId UNIQUEIDENTIFIER NOT NULL,
+  ULN BIGINT NOT NULL,
+  UKPRN BIGINT,
+  EmployerAndProviderApprovedOn DATETIME,
+  EmployerAccountId BIGINT NOT NULL,
+  CommitmentStartDate DATETIME,
+  CommitmentId BIGINT,
+  AapprenticeshipId BIGINT,
```
Comment (abroodie) [inline src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql:11]: Rename to CommitmentApprenticeshipId

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  CommitmentStatus SMALLINT,
```
Comment (abroodie) [inline src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql:12]: Should there be an FK to CommitmentStatus for this?

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  CheckType SMALLINT,
```
Comment (abroodie) [inline src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql:13]: Should there be an FK for this column?

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEmploymentVerification.sql [hunk 5]
```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [dbo].[ScheduledEmploymentVerification]
```
Comment (abroodie) [inline src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEmploymentVerification.sql:1]: Is this a duplicate of the tabled defined in ScheduledEC.sql?

**Review by abroodie — CHANGES_REQUESTED** [review]
Couple of things to look at.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/EmploymentVerification.sql [hunk 6]
```diff
@@ -1,5 +1,16 @@
 ﻿CREATE TABLE [dbo].[EmploymentVerification]
 (
-	[Id] bigint NOT NULL Identity(1,1) Constraint PK_EmploymentVerification PRIMARY KEY
+  ApprenticeshipEmploymentId BIGINT NOT NULL Identity(1,1) Constraint PK_EmploymentVerification PRIMARY KEY,
+  CorrelationId UNIQUEIDENTIFIER NOT NULL CONSTRAINT UC_EV_CorrelationId UNIQUE,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/EmploymentVerification.sql:4]: Changed.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql [hunk 7]
```diff
@@ -0,0 +1,15 @@
+CREATE TABLE [dbo].[AdhocEmploymentVerification]
+(
+  AdhocEVId BIGINT NOT NULL Identity(1,1) Constraint PK_AdhocEmploymentVerification PRIMARY KEY,
+  CorrelationId UNIQUEIDENTIFIER NOT NULL,
+  ULN BIGINT NOT NULL,
+  EmployerAccountId BIGINT NOT NULL,
+  MinDate DATETIME,
+  MaxDate DATETIME,
+  AapprenticeshipId BIGINT,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql]: Done.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql (hunk 4 above)
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql:13]: We can add. Need your suggestion. Additionally we need Unique Key to identify unique learners and enable lookback.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql [hunk 8]
```diff
@@ -0,0 +1,15 @@
+CREATE TABLE [dbo].[AdhocEmploymentVerification]
+(
+  AdhocEVId BIGINT NOT NULL Identity(1,1) Constraint PK_AdhocEmploymentVerification PRIMARY KEY,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql]: Done

File: src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+  CorrelationId UNIQUEIDENTIFIER NOT NULL,
+  ULN BIGINT NOT NULL,
+  EmployerAccountId BIGINT NOT NULL,
+  MinDate DATETIME,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql]: Changed

File: src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  MaxDate DATETIME,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql]: Changed.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/CheckType.sql [hunk 11]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE [dbo].[CheckType]
+(
+  Id INT NOT NULL Identity(1,1) Constraint PK_CheckType PRIMARY KEY,
+  CheckType VARCHAR(20),
+  CreatedOn DATETIME DEFAULT GETDATE()
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/CheckType.sql]: Removed.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/CommitmentStatus.sql [hunk 12]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE [dbo].[CommitmentStatus]
+(
+  Id INT NOT NULL Identity(1,1) Constraint PK_CommitmentStatus PRIMARY KEY,
+  CommitmentStatus VARCHAR(20),
+  CreatedOn DATETIME DEFAULT GETDATE()
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/CommitmentStatus.sql]: Removed.

File: src/SFA.DAS.EmploymentVerification.Database/Tables/LogETLCounts.sql [hunk 13]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE [dbo].[LogETLCounts]
+(
+  LogCountId BIGINT NOT NULL Identity(1,1) Constraint PK_LogETLCounts PRIMARY KEY,
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/LogETLCounts.sql]: Done

File: src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEmploymentVerification.sql (hunk 5 above)
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEmploymentVerification.sql:1]: Removed.

**Review by abroodie — CHANGES_REQUESTED** [review]
one small change

File: src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql [hunk 14]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE [dbo].[AdhocEmploymentVerification]
+(
+  AdhocEmploymentVerificationId BIGINT NOT NULL Identity(1,1) Constraint PK_AdhocEmploymentVerification PRIMARY KEY,
+  CorrelationId UNIQUEIDENTIFIER NOT NULL DEFAULT NEWID(),
```
[the remark this answers is not in the document]
Comment (supriyakolate) [reply src/SFA.DAS.EmploymentVerification.Database/Tables/AdhocEmploymentVerification.sql]: Done.

**Pull request merged**